src/main/kotlin/stringcalculator/ui/InputView.kt

```diff
@@ -0,0 +1,15 @@
+package stringcalculator.ui
+
+import java.util.Scanner
+
+val SCANNER: Scanner = Scanner(System.`in`)
+
+const val INPUT_MESSAGE: String = "수식을 입력해주세요"
+
+class InputView {
```

Comment:
객체로 만들 필요가 없는 View class는 `object` 클래스로 변경해보면 어떨까요? 😃 
```suggestion
object InputView {
```